SSO-Auth/Config/configPage.html

```diff
@@ -4,12 +4,174 @@
     <title>SSO</title>
 </head>
 <body>
-    <div data-role="page" class="page type-interior pluginConfigurationPage esqConfigurationPage">
+    <div id="sso-config-page" data-role="page" class="page type-interior pluginConfigurationPage esqConfigurationPage" data-controller="__plugin/SSO-Auth.js">
         <div data-role="content">
             <div class="content-primary">
                 <a href="https://github.com/9p4/jellyfin-plugin-sso/blob/main/README.md">Review the documentation. This plugin is configured via the API.</a>
+
+                <form id="sso-load-config" class="esqConfigurationForm">
+                    <div class="selectContainer">
+                        <label class="selectLabel" for="selectProvider">Choose OIDC Provider to load configuration for:</label>
+                        <select is="emby-select" id="selectProvider" name="selectProvider" class="emby-select-withcolor emby-select">
+    
+                        </select>
+                        <div class="selectArrowContainer"><div style="visibility:hidden;display:none;">0</div><span class="selectArrow material-icons keyboard_arrow_down" aria-hidden="true"></span></div>
+                    </div>
+
+                    <button id="LoadProvider" is="emby-button" type="button" class="raised button-submit block emby-button">
+                        <span>Load Provider</span>
+                    </button>
+
+                    <button id="DeleteProvider" is="emby-button" type="button" class="raised button-submit block emby-button">
+                        <span>Delete Provider</span>
+                    </button>
+    
+
+                </form>
+
+
+
+                <form id="sso-new-oidc-provider" class="esqConfigurationForm">
+                    <div class="verticalSection verticalSection-extrabottompadding">
+
+
+
+
+                        <div class="inputContainer">
+                            <label class="inputLabel inputLabelUnfocused" for="OidProviderName">Name/ID of OID Provider:</label>
+                            <input  is="emby-input" id="OidProviderName" required="" type="text"  class="emby-input" />
```

Comment:
I've been getting a TypeErro on these, I think I need to add a label attribute to all of the `is="emby-input"` fields

Reply:
Just returned to this, it's actually not because of a missing label attribute, its because of this https://github.com/jellyfin/jellyfin-web/issues/3592

Reply:
Removing the `class="emby-input"` should fix it